test/test_hmc.py

```diff
@@ -31,3 +32,24 @@ def test_hmc_diag_repr() -> None:
     draws_2 = np.array([hmc_2.sample()[0] for _ in range(M)])
 
     np.testing.assert_array_equal(draws_1, draws_2)
+
+
+def test_hmc_binom() -> None:
+    model = Binomial()
+    M = 800
+    hmc = HMCDiag(
+        model, stepsize=0.08, steps=3, init=np.array([model.initial_state(0)])
+    )
+
+    draws = model.constrain_draws(np.array([hmc.sample()[0] for _ in range(M)]))
+
+    # skip 100 draws to try to make estimates less noisy. e.g treat as "burn in"
```

Comment:
same comment---it's "e.g.,"   Not that minor doc comments really need to be fixed